lms/static/sass/base/_variables.scss

```diff
@@ -6,6 +6,11 @@ $baseline: 20px;
 
 // ====================
 
+// PATH
```

Comment:
OK, I'll move the paths in the cms file.  Why do we have $baseline before anything else?  Shouldn't it go in the typography section?